package.json

```diff
@@ -23,9 +23,7 @@
     "moment": "^2.10.6",
     "numeral": "^1.5.3",
     "react-addons-shallow-compare": "^0.14.7",
-    "recompose": "^0.15.1"
-  },
-  "peerDependencies": {
+    "recompose": "^0.15.1",
     "react": "^0.14.2",
```

Comment:
With the new version of NPM peerDependencies aren't being installed automatically 
https://docs.npmjs.com/files/package.json#peerdependencies

so be aware of that when making your changes. Also a lot of the tests are currently broken, I'm going to submit another PR to fix them